.studio/chef-server-collection

```diff
@@ -93,8 +93,8 @@ EOH
     "reporter": "chef-server-automate",
     "fetcher": "chef-server",
     "profiles": [{
-      "name": "apache-baseline",
-      "compliance": "test/apache-baseline"
+      "name": "ssl-baseline",
```

Comment:
I am just curious why the change from apache-baseline to ssl-baseline.

Reply:
that profile is using some deprecated resources that are causing the inspec check and exec calls to fail